.env-dist

```diff
@@ -2,6 +2,7 @@
 NODE_ENV=dev
 SERVER_URL=http://localhost:6060
 PORT=6060
+LOGOS_ORIGIN=http://localhost:6060
```

Comment:
I guess this is only a sample .env file, but what if the default value was just an empty string?
In theory that should change all these fully qualified URLs `${AppConstants.LOGOS_ORIGIN}/img/logos/${breachCard.LogoPath}` into just `/img/logos/${breachCard.LogoPath}` which seems like the behavior we'd want for localhost/Heroku, and only the stage/production configs would need to specify an ENV var for the CDN.

Although I suspect our envs currently blow up anyways if we don't add them to each Heroku environment anyways, so maybe its moot.

Reply:
Yeah there's a big issue (https://github.com/mozilla/blurts-server/issues/275) to refactor AppConfig to more gracefully handle missing env vars. But I'll change this to be an empty value - it works.